kinoshita/practice-erd/instagram/laravel/database/seeders/DatabaseSeeder.php

```diff
@@ -15,12 +15,15 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        User::factory(5) // ユーザー五人
+        User::factory()
             ->has( 
             Post::factory(2) // 全員に２投稿ずつあり
                 ->hasComments(3) // 各投稿にコメント3個ある
                 ->hasLikes(2) // 各投稿にいいね2個ある
                 ->hasTags(2) // 各投稿にたぐ2個ある
             )->create();
+
+        // 管理者
+        User::factory(1)->create(['role_id' => 0]);
```

Comment:
createは一回しか呼べず、拡張性が悪い（※）ので、stateを使いましょう。（別コメントで書いたFactory関数がベストですが）

また、`factory()`のデフォルトは1なので、省いてもいいです。
```suggestion
        User::factory()->state(['role_id' => 0])->create();
```

※例えば、nameもつけたいなら以下の双方同じstateで定義してもいいし、別々にしてもいい。間にhasなどを入れてもOK。createなら全部一個の配列で対応するしかないので、可読性の工夫などがしにくい。

```php
        User::factory()->state([
          'role_id' => 0,
          'name' => '太郎'
        ])->create();

        User::factory()
          ->state(['role_id' => 0])
          ->state(['name' => '太郎'])
        ->create();
     

```

Reply:
こちらで修正しました

[fe19b8f](https://github.com/StoryEdutech/study_repo/pull/104/commits/fe19b8f23081cedd2cbb7b9822674bc566a5955d)